src/api/spec/cassettes/BsRequestActionWebuiInfosJob/_perform/with_non_existing_target_project/1_1_2_1.yml

```diff
@@ -567,69 +25,22 @@ http_interactions:
       Connection:
       - close
       Content-Length:
-      - '135'
+      - '156'
     body:
       encoding: UTF-8
       string: |
-        <project name="home:user_2">
-          <title></title>
-          <description></description>
-          <person userid="user_2" role="maintainer" />
-        </project>
+        <status code="404">
+          <summary>project 'source_project' does not exist</summary>
+          <details>404 project 'source_project' does not exist</details>
```

Comment:
This cassette was generated without global_write_through and as such is pointless